content/en/community/_index.md

```diff
@@ -36,6 +36,12 @@ other_channels:
       Consult the [`r/grpc`](https://www.reddit.com/r/grpc/) subreddit for
       discussions related to gRPC.
 community_resources:
+ - title: >
```

Comment:
The CI is pretty unhappy with this change. My YAML is weak, but I _think_ it would be fixed by adding another space before the `- title`.

Reply:
community_resources:
  - title:

Reply:
Yes, that's the issue.